homeassistant/components/sensor/rfxtrx.py

```diff
@@ -24,6 +24,9 @@
 
 from homeassistant.const import (TEMP_CELCIUS)
 from homeassistant.helpers.entity import Entity
+import homeassistant.components.rfxtrx as rfxtrx
+from RFXtrx import SensorEvent
+from homeassistant.util import slugify
 
 REQUIREMENTS = ['https://github.com/Danielhiversen/pyRFXtrx/archive/' +
                 'ec7a1aaddf8270db6e5da1c13d58c1547effd7cf.zip#RFXtrx==0.15']
```

Comment:
Effectivelly, the latest pyRFXtrx version is needed. @Danielhiversen can
you make a pyRFXtrx release (github) and send me a URL ?

Thanks

On Tue, Oct 6, 2015 at 11:43 AM, Daniel Høyer Iversen <
notifications@github.com> wrote:

> In homeassistant/components/sensor/rfxtrx.py
> <https://github.com/balloob/home-assistant/pull/483#discussion_r41244492>:
>
> > @@ -24,6 +24,9 @@
> >
> >  from homeassistant.const import (TEMP_CELCIUS)
> >  from homeassistant.helpers.entity import Entity
> > +import homeassistant.components.rfxtrx as rfxtrx
> > +from RFXtrx import SensorEvent
> > +from homeassistant.util import slugify
> >
> >  REQUIREMENTS = ['https://github.com/Danielhiversen/pyRFXtrx/archive/' +
> >                  'ec7a1aaddf8270db6e5da1c13d58c1547effd7cf.zip#RFXtrx==0.15']
>
> @badele <https://github.com/badele> : Should this be updated to latest
> version of the library?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/483/files#r41244492>.
>



-- 

------------------------------

*Bruno Adele*
bruno.adele@jesuislibre.org
Perso: jesuislibre <http://www.jesuislibre.org> | blog
<http://blog.jesuislibre.org> | twitter <http://twitter.com/jesuislibre> |
flickr <http://www.flickr.com/photos/b_adele>
Pro: cendreo <http://www.cendreo.com> | blog <http://blog.jesuislibre.org>
Site : informemoi.com